pkg/oonimkall/session.go

```diff
@@ -124,7 +127,6 @@ func NewSession(config *SessionConfig) (*Session, error) {
 		})
 	}
 	engineConfig := engine.SessionConfig{
-		AssetsDir:              config.AssetsDir,
```

Comment:
Maybe we should continue propagating `AssetsDir` and cleanup the assets dir in the session?

Reply:
No, I've taken another approach